.editorconfig

```diff
@@ -11,5 +11,5 @@ trim_trailing_whitespace = true
 [*.md]
 trim_trailing_whitespace = false
 
-[*.{yml,yaml}]
+[*.{yml,yaml,stub}]
```

Comment:
Can you please revert this change? It's inconsistent with other packages. Not all stub files are yaml files.